src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/EventHandlerModifier/EventHandlerModifierTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Acuminator.Analyzers.StaticAnalysis;
+using Acuminator.Analyzers.StaticAnalysis.PrivateEventHandlers;
+using Acuminator.Analyzers.StaticAnalysis.PXGraph;
+using Acuminator.Tests.Helpers;
+using Acuminator.Tests.Verification;
+using Acuminator.Utilities;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Xunit;
+
+namespace Acuminator.Tests.Tests.StaticAnalysis.EventHandlerModifier
```

Comment:
I don't mind separate classes for diagnostic and code fix tests, But it's also normal to have a single test class to check both code fix and diagnostic.

Reply:
I didn't do anything for this at the moment. If you have anything concrete in mind, or if you want to point to a reference test, please let me know.